applications/ConstitutiveLawsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.cpp

```diff
@@ -42,6 +42,109 @@
 
 namespace Kratos
 {
+template <class TConstLawIntegratorType>
+void GenericSmallStrainHighCycleFatigueLaw<TConstLawIntegratorType>::InitializeMaterialResponseCauchy(ConstitutiveLaw::Parameters& rValues)
+{
+    double max_stress = mMaxStress;
```

Comment:
this huge initialization list can be encapsulated but we'll do it with a structure in the future

Reply:
Yep

Reply:
And if it can const better

Reply:
> And if it can const better

Only two variables allow const right now. I will think about this while implementing the structure, as this part can be probably optimized.